ppr-ui/src/composables/useRegistration.ts

```diff
@@ -108,6 +108,18 @@ export const useRegistration = () => {
     return false
   }
 
+  const hasRenewal = (
```

Comment:
Is this checking if there has been a previous renewal registration?

Reply:
yep. it goes through the children to see if there has been a previous renewal

Reply:
So should it check that registrationClass === 'RENEWAL' or registrationType === 'RE'?

Reply:
yes, good point. I've made the fix